docs/Flattening.md

```diff
@@ -96,6 +98,6 @@ dto.CustomerName.ShouldEqual("George Costanza");
 dto.Total.ShouldEqual(74.85m);
 ```
 
-We configured the type map in AutoMapper with the CreateMap method.  AutoMapper can only map type pairs it knows about, so we have explicitly register the source/destination type pair with CreateMap.  To perform the mapping, we use the Map method.
+We configured the type map in AutoMapper with the **_CreateMap_** method.  AutoMapper can only map type pairs it knows about, so we have explicitly register the source/destination type pair with CreateMap.  To perform the mapping, we use the **_Map_** method.
```

Comment:
Should do ticks for inline code (`) instead of bolded etc.

Reply:
... "so we have explicitly register" ... -> ... "so we have to explicitly register "...